INSTALL.rst

```diff
@@ -16,68 +16,46 @@ environment.
 -------------------------
 
 ``scikit-image`` comes pre-installed with several Python distributions,
-including Anaconda_, `Enthought Canopy`_, `Python(x,y)`_ and `WinPython`_.
-However, you can install or upgrade existing ``scikit-image`` via
-shell/command prompt.
+including Anaconda_, `Enthought Canopy`_, `Python(x,y)`_ and
+`WinPython`_.
 
-a. Windows
-``````````
-
-On Windows, you can install ``scikit-image`` using::
-
-    pip install scikit-image
-
-For Conda-based distributions (Anaconda, Miniconda), execute::
-
-    conda install scikit-image
+On all other systems, install it via shell/command prompt::
 
-If you are using pure Python i.e. the distribution from python.org_, you'll
-need to manually download packages (such as numpy, scipy and scikit-image)
-using Python wheels available from `Christoph Gohlke's`_ website.
-You can install Python wheels using::
-
-    pip install SomePackage-1.0-py2.py3-none-any.whl
-
-.. _Anaconda: https://store.continuum.io/cshop/anaconda/
-.. _Enthought Canopy: https://www.enthought.com/products/canopy/
-.. _Python(x,y): http://python-xy.github.io/
-.. _WinPython: https://winpython.github.io/
-
-b. Debian and Ubuntu
-````````````````````
+  pip install scikit-image
 
-On Debian and Ubuntu, install ``scikit-image`` with::
+If you are running Anaconda or miniconda, use::
 
-  sudo apt-get install python-skimage
```

Comment:
Is this no longer the case for Debian? Did they remove the package from the repositories?

Reply:
Yes, I suspect that is what the greater majority of Debian users want.  Those who don't probably also know how to apt-get.